packages/gui/src/components/menu-bar/menu-bar.css

```
@@ -33,6 +33,24 @@
     background-color: $menu-bar-background;
     background-image: $menu-bar-background-image;
     color: $menu-bar-foreground;
+
+    transition: background 0.35s, margin 0.35s;
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)